experiments/mjc_mdgps_ioc_example/hyperparams.py

```diff
@@ -0,0 +1,284 @@
+""" Hyperparameters for MJC peg insertion policy optimization. """
```

Comment:
we don't have a peg example in the paper so we can remove this file from the PR.